jstz_cli/src/repl.rs

```diff
@@ -110,19 +110,32 @@ fn evaluate(input: &str, rt: &mut Runtime, hrt: &mut (impl HostRuntime + 'static
         Ok(value)
     });
 
-    match rt_output {
-        Ok(res) => {
-            if !res.is_undefined() {
-                println!(
-                    "{}",
-                    res.to_string(&mut rt.context())
-                        .unwrap()
-                        .to_std_string_escaped()
-                );
+    jstz_core::future::block_on(async move {
+        match rt_output {
+            Ok(mut res) => {
+                if res.is_promise() {
+                    if let Ok(promise_result) = rt.resolve_value(&res).await {
+                        res = promise_result;
+                    }
+                }
```

Comment:
`resolve_value` will handle the `is_promise` logic. Can you move this to where we define `rt_output`